ideas42ventures/ideas42ventures.com
# Added 404 page

Works on #27 

- Added a 404 redirect.


**How to test:** 

1. The links below should lead to valid URLs with their respective content on each page.

[Invalid URL](https://deploy-preview-30--ideas42ventures.netlify.app/2) should render a [404 page](https://deploy-preview-30--ideas42ventures.netlify.app/404.)
